src/Console/Commands/ImportData.php

```diff
@@ -121,7 +154,21 @@ protected function save(string $model, array $data)
     }
 
     /**
-     * @param CsvData $data
+     * @param class-string<SubnetV4|SubnetV6> $model
+     * @param AsnCsvData                      $data
+     *
+     * @return SubnetV4|SubnetV6
+     */
+    protected function saveAsn(string $model, array $data)
+    {
+        $asn = Asn::updateOrCreate(['id' => $data['asn']], ['title' => $data['as']]);
+
+        return $model::where(['network_start' => $data['network_start'], 'network_end' => $data['network_end']])
```

Comment:
if you compare two files and db11.csv and asn.csv you can see network_start and network_end is not the same in both files.
![image](https://user-images.githubusercontent.com/16887332/213139752-bea6df86-567c-45cf-864b-34c5e973a704.png)

_left is db11_ipv4.csv and other is asn_ipv4.csv_

Reply:
i have some changes in this method , please review that